src/pamiq_core/console/cui.py

```diff
@@ -0,0 +1,141 @@
+import argparse
+import cmd
+import json
+from typing import override
+
+import requests
```

Comment:
`httpx`を使いたい。`requests`は古いライブラリで、型アノテーションがなかったりする。`Starlette`をWebApiHandlerに使っており、これと同じ開発元でかつ高速で非同期処理も可能な`httpx`が相性的にも今後開発をしていく上でも良いと思う。

ちなみに使い方は`requests`とほぼ同じ

```suggestion
import httpx
```

Reply:
https://github.com/MLShukai/pamiq-core/pull/210/commits/6d492c7d9a19fa42de310ad9a6512368f348ff59 とhttps://github.com/MLShukai/pamiq-core/pull/210/commits/8bbd8b402ae0c7c22f7fea35799e5ed2fceb2cc0 で`httpx`に変更